src/panel/category_panel.js

```diff
@@ -111,10 +114,9 @@ export default class CategoryPanel {
   }
 
   selectPoi(poi){
-    // TODO telemetry
     fire('fit_map', poi, layouts.LIST)
     this.close(false)
-    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this})
+    PanelManager.loadPoiById(poi.id, {isFromList : true, list: this, source: constants.telemetry.source.PJ})
```

Comment:
`source` should be read from the API response. The category panel may be filled with POIs from different sources (depending on the location, the category, etc.)

Reply:
Ok @amatissart, I don't see any `source` attribute coming from the API response in the IdunnPOI object. I don't think extracting it from the `id` would be a sustainable solution.